chapters/intro/main.tex

```diff
@@ -1,26 +1,165 @@
 \chapter{Introduction}
 \label{chp:intro}
 
-This will be the introduction.
+The original objective of this thesis was to utilise machine learning to
+better understand variability in the National Health Service (NHS). Driven by
+the needs of the co-sponsors of this project, the Cwm Taf Morgannwg University
+Health Board (UHB), the hope was to apply some technique(s) from machine
+learning to reveal insights into their patient population within the hospital
+system. The journey to this point has not been without its deviations from this
+primary task, but that objective has been achieved.
+
+With an administrative dataset provided by \ctmuhb, exploratory analysis found
+that the population in question was deeply varied and heterogeneous --- as
+expected. However, the dataset was insufficiently detailed to construct
+meaningful models on the entire population or system, opening up an interest in
+population segmentation techniques. While data requests were made, the gathering
+of literature began. Even with the expansive nature of machine learning
+literature, two clear patterns emerged.
```

Comment:
This is good but perhaps could be worded in a different way. Perhaps a more generic: "common problems in applying machine learning to healthcare are ... the contribution of this thesis is to use population segmentation ..."